finetune/target_models/masked_language_model.py

```diff
@@ -62,14 +62,18 @@ def text_to_tokens_mask(self, X, Y=None, context=None):
                     ]
                 )
             ] = False
+            mlm_positions = np.where(mlm_mask)[0]
+            
+            if len(mlm_positions) > self.config.max_masked_tokens: # subsample
+                np.random.shuffle(mlm_positions) # means we don't bias the begining of the sequence
+                mlm_positions = mlm_positions[:self.config.max_masked_tokens]
```

Comment:
nitpick, but probably more efficient to do something like this?

positions_to_keep = np.random.choice(range(seq_len), size=self.config.max_masked_tokens)
mlm_positions = mlm_positions[positions_to_keep]